app/assets/stylesheets/puppetclasses.scss

```diff
@@ -32,8 +32,12 @@
   .matcher .matcher-group {
     display: block;
   }
-  
+
   .matcher_value {
     width: 60%;
   }
 }
+
+.selected_puppetclass.unavailable a {
+  color: $gray-lighter;
```

Comment:
Use something from http://www.patternfly.org/styles/color-palette/#_ , the current one is barely visible

`pf-black-400 #bbbbbb` looks nice IMO for disabled elements